clusterloader2/util/unmarshaling.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/viper"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/kubernetes/pkg/api/legacyscheme"
+	"k8s.io/perf-tests/clusterloader2/api"
+)
+
+// ReadConfig creates config from file specified by the given path.
+func ReadConfig(path string) (*api.Config, error) {
```

Comment:
Since what you need is name of config - let's pass a name here. If needed we can have a wrapper to translate from path.

Reply:
I think that passing path is better that passing only a name. I assume that user will pass path to config, which can be anywhere.